Gruntfile.js

```diff
@@ -24,7 +24,8 @@ module.exports = function(grunt) {
 		// removes all distrubtions prior to rebuilding
 		//
 		'clean': {
-			all: [DIST_OPTIMIZED, DIST_ANDROID, DIST_IOS, DIST_SPRITE, DOC_DEST]
+			all: [DIST_OPTIMIZED, DIST_IOS, DIST_SPRITE, DOC_DEST],
+			icons: [DIST_ICONS_IOS, DIST_ICONS_WEB]
```

Comment:
we should probably keep this at one clean task so that the default grunt build will:

1. clean all dists
2. build all dists